Client/Frontend/Browser/Playlist/VideoPlayer/VideoPlayerInfoBar.swift

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import AVKit
+
+class VideoPlayerInfoBar: UIView {
+    private let controlStackView = UIStackView().then {
+        $0.axis = .horizontal
+        $0.spacing = 32.0
+    }
+    
+    private let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark)).then {
+        $0.contentView.backgroundColor = #colorLiteral(red: 0.231372549, green: 0.2431372549, blue: 0.3098039216, alpha: 0.8)
+    }
+    
+    private let favIconImageView = UIImageView().then {
+        $0.contentMode = .scaleAspectFit
+        $0.layer.cornerRadius = 6.0
+        $0.layer.masksToBounds = true
```

Comment:
Can add continous curve to iOS13+
https://developer.apple.com/documentation/quartzcore/calayercornercurve/3152600-continuous